examples/aws-values.yaml

```diff
@@ -1,7 +1,10 @@
-pachd: 
+pachd:
   storage:
     backend: AMAZON
     amazon:
       amazonBucket: blah
-      amazonSecret: blah
-      amazonID: blah
+      # this is an example access key ID from https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html
+      id: AKIAIOSFODNN7EXAMPLE
+      # this is an example secret access key (ibid.)
```

Comment:
ibid?

Reply:
Short way of saying ‘also found in https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html’ ☺

But I can spell it out too. Easy change *grin*